src/lib.rs

```diff
@@ -232,7 +234,7 @@ pub use self::client::{Client, ClientBuilder};
 pub use self::error::{Error, Result};
 pub use self::body::Body;
 pub use self::into_url::IntoUrl;
-pub use self::proxy::Proxy;
+pub use self::proxy::{Proxy, get_proxies};
```

Comment:
Do we need to publicly expose `get_proxies`? I'd prefer to keep it out of the public API...

Reply:
I think it's a useful `util` function to get system proxy setting.  So I make it public.
But I'm ok to make `get_proxies` private :)